.goreleaser.yml

```diff
@@ -41,7 +41,7 @@ builds:
     hooks:
       post:
         - cmd: >-
-            {{ if eq .Runtime.Goos "windows" }}.\script\sign{{ else }}./script/sign{{ end }} '{{ .Path }}'
+            {{ if eq .Runtime.Goos "windows" }}pwsh .\script\sign.ps1{{ else }}./script/sign{{ end }} '{{ .Path }}'
```

Comment:
Calling out that `script/sign.ps1` is carried over from the prototyping as-is, so don't be alarmed by all of the `script/sign*` deletion in the PR 😅